docs/get-started/labs/lab-34.md

```diff
@@ -0,0 +1,70 @@
+---
+slug: lab-34
+title: Use Sealed Secrets
+sidebar_label: Create RabbitMQ cluster
+---
+
+In this lab we will create sealed secrets and see how they can be publicly stored in a git repository.
```

Comment:
how they can be publicly stored in a git repository. => how to securely sensitive information in git repository